### PR TITLE
Add action for greeting first-time contributors [skip netlify] [skip ci]

### PR DESCRIPTION
**Pull request by trsvchn**
New Feature

Description:

This PR welcomes first-time contributors with issue or PR comment.

Current welcome message includes very short info and link to Contributing guide. Feel free to propose what needs to be included or changed or formatted (we can use markdown here).

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

**Comment by vfdev-5** [comment]
Thanks for this idea @trsvchn ! For the first issue/PR, let's use that, LGTM.
Even if we could automate some interaction and messaging, let's do not forget that community and human interaction is the most important here, I think.

**Review by vfdev-5 — APPROVED** [review]
Thanks @trsvchn !

**Comment by trsvchn** [comment]
> Even if we could automate some interaction and messaging, let's do not forget that community and human interaction is the most important here, I think.

Totally agree! The aim is NOT to replace or resuce live interaction with the community but to guide new contributors. :)

File: .github/workflows/welcome.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+on: [issues, pull_request]
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          issue-message: "**Thank you for opening your First Issue!**\nIf you would like to help us with the code, please feel free to discuss it here, take a look at our [Contributing Guide](https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md), and raise a PR."
```
Comment (vfdev-5) [inline .github/workflows/welcome.yml:12]: I think we can add some irrelevant info which could be ignored all together.
For example, user opens an issue as a question and we say to "help with the code" and "raise a PR"...

I think we could remain clear and synthetic:
```
**Thank you for opening your First Issue!**

We appreciate a lot user's feedback on what we are doing!

If you'd like to contribute to the project, please check out our [Contributing Guide](https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md).
```
What do you think ?

Comment (trsvchn) [reply .github/workflows/welcome.yml:12]: Yes, that make sense!

File: .github/workflows/welcome.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+          pr-message: "**Thank you for opening your First Pull Request!**\nPlease take a look at our [Contributing Guide](https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md)"
```
Comment (vfdev-5) [inline .github/workflows/welcome.yml:13]: The phrase `Please take a look at our Contributing Guide` may be interpreted as user does something wrong ?

Comment (trsvchn) [reply .github/workflows/welcome.yml:13]: yeah, I was thinking to use this action only for first-issues?

Comment (vfdev-5) [reply .github/workflows/welcome.yml:13]: Yes, I understand that. I mean I'd like to find some wording saying that it is good optionally to check out the contributing guide and not imperatively..

File: .github/workflows/welcome.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+on: [issues, pull_request]
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          issue-message: "**Thank you for opening your First Issue!**\n\nWe appreciate a lot user's feedback on what we are doing!\n\nIf you'd like to contribute to the project, please check out our [Contributing Guide](https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md)."
+          pr-message: "**Thank you for opening your First Pull Request!**\nPlease take a look at our [Contributing Guide](https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md)"
```
Comment (vfdev-5) [inline .github/workflows/welcome.yml:12]:
```suggestion
          pr-message: "**Thank you for opening your First Pull Request!**\nWe appreciate a lot community contributions as pull requests!\n\nIf you would like to get more details on the project development, please take a look at our [Contributing Guide](https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md)."
```

**Review by vfdev-5 — APPROVED** [review]
Thanks @trsvchn for the idea and the PR !

**Comment by trsvchn** [comment]
Ok, I think it failed to run here: issue 1822 because of yet another `pull_request` / `pull_request_target` story. And another thing that I annoys me is  its not possible to hide the run, I was expecting it running silently, but we have this workflow in a check list on every PR.

I'm considering to merge `welcome` and `pr-labeler` into one `triage` workflow. @vfdev-5 what do you think?

**Comment by vfdev-5** [comment]
@trsvchn actually, where do you see it failling ?
I'm also a bit worried about multiplication of actions we have now... 
Yes, we can merge it with pr-labeler.

**Comment by trsvchn** [comment]
@vfdev-5 It was before the force-update
https://github.com/pytorch/ignite/runs/2142060380?check_suite_focus=true#step:3:33

**Comment by vfdev-5** [comment]
Btw, this PR is the first contribution but action didn't work: https://github.com/pytorch/ignite/pull/1822

**Comment by trsvchn** [comment]
yeah, It requires `pull_request_target` (I think)

**Pull request merged**